base/applications/notepad/lang/tr-TR.rc

```diff
@@ -1,5 +1,6 @@
 // COPYRIGHT: 2012 Arda Tanrıkulu <ardatanrikulu@gmail.com>
 //            2013-2019 Erdem Ersoy
+//            2024-Mahir Gül mahirgul@gmail.com
```

Comment:
Stasm already requested that minus to be stripped earlier as part of his suggestion. But since you didn't apply his suggestion, and edited the line manually instead, it is still not fixed:
```suggestion
//            2024 Mahir Gül mahirgul@gmail.com
```

Reply:
I think I'll just update the header entirely to our standards.